ui/layout-pages-fixed.md

```diff
@@ -0,0 +1,377 @@
+
+```xml
+<svg width="210mm" height="297mm" ... some information omitted here>
+<defs id="defs1" />
+  <g inkscape:label="Layer 1" inkscape:groupmode="layer id="layer1">
+    <rect style="fill:#0000ff;fill-rule:evenodd;stroke-width:0.264583"
+       id="rect1"
+       width="36.495762"
+       height="25.169491"
+       x="30.203388"
+       y="31.461863" />
+    <rect style="fill:#ff2a00;fill-rule:evenodd;stroke-width:0.264583;fill-opacity:1"
+        id="rect2"
+       ...omitted />
+    <rect style="fill:#0000ff;fill-rule:evenodd;stroke-width:0.264583"
+        id="rect3"
+    ...omitted />
+  </g>
+</svg>
+```
+
+You can see several elements that represent the rectangles we have drawn.
+To make the SVG interactive, you need to add a special attribute to the elements you want to be clickable and therefore interactive.
+This attribute is called `openhab` and can be set to any value, but it is recommended to use a meaningful value like "true".
+
+Let's add this attribute to the first rectangle either via Inkscape or directly within the file, then save the image and reload the page:
+
+![interactive-element-add-openhab.png](images/interactive-element-add-openhab.png)
+
+```xml
+  <rect style="fill:#0000ff;fill-rule:evenodd;stroke-width:0.264583"
+    id="rect1"
+    width="36.495762"
+    height="25.169491"
+    x="30.203388"
+    y="31.461863"
+    openhab="true" />
+```
+
+Note that you have added the `openhab="true"` attribute to the first `<rect>`angle  and not to a group `<g>` (which we talk about below)
+
+- Make sure that the `Embed SVG` option is enabled in the Fixed Canvas Layout configuration.
+- Stay in Edit-Mode of the page
+- On the right between the "Grid" icon and the "Configure Layout" icon, you can see a new icon with a flash which allows you to flash the interactive elements.
+When you click on it, the first rectangle should flash which indicates that has become an interactive element for openHAB.
+- There are two different colors when the element flashes:
+  - Red: The element is defined as an interactive element but is not yet configured.
+  - Green: The element is defined as an interactive element and is already configured.
+- Click on it and a dialog, which is similar to configurations of a normal widget except that it has some additional settings, will appear.
+- Later, after configuration, you can go to Run-Mode and try out your configuration.
+
+### Configuration of an interactive element
+
+The dialog has many options that allow you to configure the interactive element.
+Usually you will first configure the Action and the Action Item like in a normal widget.
+
+![interactive-element-action.png](images/interactive-element-action.png)
+
+Many special options though will only be available when you click on "Show Advanced".
+
+- **State Items**: Define the item that is used to retrieve the state of the element to reflect the state.
```

Comment:
I think it should be mentioned that this defaults to the action Item (IIRC, but the code should be checked to be sure).